modules/ROOT/pages/clustering/setup/analytical.adoc

```diff
@@ -0,0 +1,268 @@
+
+[source, cypher, role=noplay]
+----
+SHOW SERVERS YIELD *;
+----
+
+[queryresult]
+----
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| serverId                               | name                                   | address          | state     | health      | hosting             | requestedHosting    | tags | allowedDatabases | deniedDatabases | modeConstraint | version     |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "f3bd1199-bc6f-4a38-b25c-5f7588df5182" | "f3bd1199-bc6f-4a38-b25c-5f7588df5182" | "server001:7687" | "Enabled" | "Available" | ["system", "neo4j"] | ["system", "neo4j"] | []   | []               | []              | "NONE"         | "5.8.0"     |
+| "b331e481-c2ba-4b4e-82f3-bb51fe171483" | "b331e481-c2ba-4b4e-82f3-bb51fe171483" | "server002:7687" | "Enabled" | "Available" | ["system"]          | ["system"]          | []   | []               | []              | "NONE"         | "5.8.0"     |
+| "bd80e8fd-a51b-406a-9ed4-42daf4792aa6" | "bd80e8fd-a51b-406a-9ed4-42daf4792aa6" | "server003:7687" | "Enabled" | "Available" | ["system"]          | ["system"]          | []   | []               | []              | "NONE"         | "5.8.0"     |
+| "df3758b1-337f-4b8a-a9de-8e745ca96549" | "df3758b1-337f-4b8a-a9de-8e745ca96549" | "server004:7687" | "Free"    | "Available" | ["system"]          | []                  | []   | []               | []              | "SECONDARY"    | "5.8.0"     |
+| "9207bfd9-aa1b-40c2-b965-edcd3955a20e" | "9207bfd9-aa1b-40c2-b965-edcd3955a20e" | "server005:7687" | "Free"    | "Available" | ["system"]          | []                  | []   | []               | []              | "SECONDARY"    | "5.8.0"     |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+====
+
+[[cluster-example-create-databases-on-cluster]]
+=== Create new databases in the cluster
+
+As mentioned in the xref:clustering/introduction.adoc[Introduction], a server in a cluster can either host a database in primary or secondary mode.
+The primaries provide write operations and fault tolerance if there are more than one of them.
+The secondaries provide somewhere for read queries to run, including the analytical queries for this example.
+
+.Create a new database with one primary and two secondaries
+====
+In the `system` database on the writer from the previous example, execute the following Cypher command to create a new database:
```

Comment:
There is no _writer_ specified in the previous example. Do you mean to say to execute on any of the primaries? They are essentially all writers, right?

Reply:
Yep, that's pretty much a copy-paste failure. I'll just say 'on the `system` database'.